counterpartylib/test/conftest.py

```diff
@@ -23,38 +23,59 @@
 
 # we swap out util.enabled with a custom one which has the option to mock the protocol changes
 MOCK_PROTOCOL_CHANGES = {
-    'bytespersigop': False  # default to False to avoid all old vectors breaking
+    'bytespersigop': False,    # default to False to avoid all old vectors breaking
 }
 MOCK_PROTOCOL_CHANGES_AT_BLOCK = {
-    'subassets': {'block_index': 310495, 'enabled': True},  # override to be true only after block 310495
+    'subassets': {'block_index': 310495, 'allow_always_latest': True},  # override to be true only at block 310495
+    'short_tx_type_id': {'block_index': 310502, 'allow_always_latest': False},  # override to be true only at block 310502
```

Comment:
for these protocol changes, we want to test the effects before and after certain blocks.